### DISCUSSION TITLE
FR: Ability to show cart count in more than one element

### DISCUSSION THREAD

**Discussion by sunscreem** [statamic-rad-pack/shopify#247] [Feature Requests]
We've got a shopping cart icon with the cart count in two places in our code, one for desktop, one for mobile. It would have handy if the plug-in could support multiple elements, at present it just targets the first one it finds.

**Comment by ryanmitchell** [comment]
The whole approach is being changed in v5 which is going to be tagged, we've moved to an Alpine helper

**Reply by ryanmitchell** [reply]
See https://statamic-shopify-docs.vercel.app/frontend/storefront-api
